client/widgets/Buttons.h

```diff
@@ -191,8 +190,29 @@ class CToggleGroup : public CIntObject
 	void addToggle(int index, CToggleBase * button);
 	/// Changes selection to specific value. Will select toggle with this ID, if present
 	void setSelected(int id);
+};
+
+/// A typical slider for volume with an animated indicator
+class CVolumeSlider : public CIntObject
+{
+	int value;
+	CFunctionList<void(int)> onChange;
+	CAnimation * const anim;
+	CAnimImage * const image;
```

Comment:
Feel free to add accessor for anim field in  CAnimImage and leave only one pointer here